### ISSUE TITLE
Adjust minimum required Vue version

### ISSUE THREAD

**Issue by Techassi** [vue-youtube/vue-youtube#14]
You have bumped the dependencies, as I see from package.json. While this is generally a good idea, I now get dependency issues when building my project.

```
D:\masu-data\github-suterma\replayer-pwa [main ≡]> npm i   
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: replayer@2.0.0
npm ERR! Found: @vue-youtube/core@0.0.4
npm ERR! node_modules/@vue-youtube/core
npm ERR!   @vue-youtube/core@"^0.0.6" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! @vue-youtube/core@"^0.0.6" from the root project  
npm ERR!
npm ERR! Conflicting peer dependency: vue@3.3.7
npm ERR! node_modules/vue
npm ERR!   peer vue@">= 2.5 < 2.7" from @vue/composition-api@1.7.2
npm ERR!   node_modules/@vue/composition-api
npm ERR!     peerOptional @vue/composition-api@"^1.7.2" from @vue-youtube/core@0.0.6
npm ERR!     node_modules/@vue-youtube/core
npm ERR!       @vue-youtube/core@"^0.0.6" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
This is because my project still uses ` "vue": "^3.2.47",` It should not be a problem for me, more of a gentle reminder to bump vue at my side too. However, some people might not be able or willing to change their other dependencies.

I suggest you reverse your vue dependency to some lower version, mabe even just 3.0.0?

_Originally posted by @suterma in https://github.com/vue-youtube/vue-youtube/issues/8#issuecomment-1782310917_

**Comment by suterma** [comment]
For me personally, this issue is resolved by me changing to vite now, and using "vue": "^3.3.4" now. But for others, coming from older vue releases, this might still be an issue.

**Comment by Techassi** [comment]
Alright, good to know. I will look into decreasing the minimum required version slightly. I will aim for 1+ year of backwards compatibility. If users need even older versions, we can revisit our decision.

**Comment by Techassi** [comment]
I would consider this as resolved, because Vue `3.3.7` was released in October 2023. This release is now roughly one year old.

Feel free to re-open this issue if you have any objections.

**Comment by suterma** [comment]
I am now at vue 3.4.15 and @vue-youtube/core 0.0.6. Everything is working great!
